server/game/cards/09.1-FtE/CommandRespect.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes, Players } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class CommandRespect extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'takes honors from opponent when they play an event',
+            condition: context => {
+                return context.player.hand.size() < context.player.opponent.hand.size() &&
```

Comment:
Should check for context.player.opponent to not be null to avoid a null reference